tests/integration/test_service_dev.py

```diff
@@ -2,9 +2,11 @@
 This file contains integration tests. We use the CLI to exercise functionality of
 various DFFML classes and constructs.
 """
+import io
+import json
 import inspect
 import pathlib
-
+import unittest
```

Comment:
I accidentally removed these when I went through and removed all unused import. The CI wasn't failing... weird. I figured out what was wrong: https://github.com/intel/dffml/issues/358

If you merge master into this branch we'll be good to go.